src/components/registration/steps.jsx

```diff
@@ -91,6 +91,13 @@ module.exports = {
         },
         validateUsername: function (username, callback) {
             callback = callback || function () {};
+            if (username.length < 1) {
```

Comment:
`if (!username)` might be safer since it would cover null/undefined too (and would avoid a possible "Cannot read property 'length' of undefined" error).